### DISCUSSION TITLE
`[up-follow]` and Bootstrap 5

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#565] [Q&A] [answered]
Hi,

I know Bootstrap is not Unpoly's business but I would like to ask for some advice: I have a BS dropdown where each item is a link with `up-follow`. Apparently, `up-follow` captures the click event preventing the dropdown to auto-close on click.

I've tried some solutions to close the dropdown programmaticaly after the click with no success. 

Any tips would be appreciated.

TIA.

**Comment by triskweline** [comment, marked as answer]
Maybe something like this could work?

```js
up.on('up:link:follow', (event, link) => {
  bootstrap.Dropdown.getOrCreateInstance(element).hide()
})
```

**Reply by alexandremjacques** [reply]
Hi!

I ended up using a variation of your suggestion and solved this way:

```js
up.on('up:link:follow', (event, link) => {
    let dd = up.fragment.closest(link, '.dropend')
    bootstrap.Dropdown.getOrCreateInstance(dd).toggle()
})
```

The dropdown element is not the `link` itself - it's its container with the `.dropend` class. So, I had to search for it. For reference, I'm using an `.dropend` (simulating a right-side drop-menu), not a `.dropdown` (the default behaviour).

Thanks again!!
